moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -208,20 +208,22 @@ export default function TheMap({
     setBounds(newBounds.flat());
   };
 
-  const componentFeatureCollection =
-    useComponentFeatureCollection(clickedComponent);
+  // This is a temporary to get data into the map sources
+  const data = {
+    draftComponentFeatures,
+    ctnLinesGeojson,
+    ctnPointsGeojson,
+    projectLines,
+    projectPoints,
+  };
+
+  const interactiveLayerIds = makeInteractiveIds();
```

Comment:
i was going to suggestion memoizing this, but, since it's completely static, you might just declare `interactiveLayerIds` in `mapUtils` and import it?

Reply:
Just made this change because I totally agree 👍